lhotse/parallel.py

```diff
@@ -73,3 +77,69 @@ def run(self) -> None:
             for args in zip(*self.iterables):
                 future = ex.submit(self.fn, *args)
                 self.queue.put(future, block=True)
+
+
+class ParallelExecutor:
+    """
+    A class which uses ProcessPoolExecutor to parallelize the execution of a callable class.
+    The instances of the runner class are instantiated separately in each worker process.
+    """
```

Comment:
Coud you add an example of the usage in this doc? This API is not self-explanatory to me.